libs/server/Custom/CustomCommandManager.cs

```diff
@@ -13,26 +14,35 @@ namespace Garnet.server
     /// </summary>
     public class CustomCommandManager
     {
+        // Maximum number of custom raw string commands
         private static readonly int MaxCustomRawStringCommands = 256;
 
+        // Initial size of expandable maps
+        private static readonly int MinMapSize = 8;
```

Comment:
why not start the expandable map at 0 so that we dont pay the cost of memory in the common case that there are no custom commands? then then grow as 1,2,4, etc. (for CCM and CCMS)